plugin/src/test/java/org/owasp/benchmarkutils/score/parsers/VeracodeReaderTest.java

```diff
@@ -0,0 +1,23 @@
+package org.owasp.benchmarkutils.score.parsers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.owasp.benchmarkutils.score.BenchmarkScore;
+import org.owasp.benchmarkutils.score.ResultFile;
+import org.owasp.benchmarkutils.score.TestHelper;
+
+class VeracodeReaderTest extends ReaderTestBase {
+
+    private ResultFile resultFile;
+
+    @BeforeEach
+    void setUp() {
+        resultFile = TestHelper.resultFileOf("testfiles/Benchmark_Veracode.xml");
+        BenchmarkScore.TESTCASENAME = "BenchmarkTest";
+    }
+
+    @Test
+    void onlyVeracodeReportCanReadAsTrue() {
+        assertOnlyMatcherClassIs(this.resultFile, VeracodeReader.class);
+    }
```

Comment:
There is an issue with the ReaderTestBase file. When I try to change the condition check logic to checking the root node name, the ReaderTestBase is always throwing "FileNotFound" exception. I have tested the reader by building it, was able to generate the scorecard. So, I am removing the test for this Reader alone.

Reply:
Yes, this is because of the way the reader accesses the data. In VeracodeReader, please change

InputSource is = new InputSource(new FileInputStream(resultFile.file()));

to

InputSource is = new InputSource(new StringReader(resultFile.content()));

Reply:
Figured it and solved. will push the code